constants/column_data_type.go

```diff
@@ -0,0 +1,48 @@
+package constants
+
+const (
+	ColumnDataTypeBoolean   = "boolean"
```

Comment:
maybe rename these DuckDbTypeXXXX ? these are basically valid duck db types used to build the queries.  A bit less verbose